zed/zed-opencv/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
```

Comment:
Why are we using Stereolabs' license and not our own NCSA one? Is this because of the reference code that you used?

Reply:
I can change the directory structure. The Stereolab's license is from the reference code (their OpenCV sample).

Reply:
Did we use that code in our code? If we did then we have to keep the Stereolab License.

Reply:
I used their function of shared pointers between ZED images and their OpenCV counterparts.